main.go

```diff
@@ -22,6 +22,7 @@ import (
 	"os"
 	"runtime"
 
+	"github.com/kuadrant/kuadrant-operator/pkg/common"
```

Comment:
nitpick: I fail to see the logic behind how we group these dependencies. Maybe in another PR we can work on this and give them some order people can reason about. E.g. Kuadrant-controlled packages always grouped together, or layered based on proximity to the Golang std lib.

Reply:
Yeah, grouping is inconsistent in a few places. I think the main reason, at least for me, for this inconsistency is that these are frequently auto added by an IDE but ignores the groups that have been already been logically broken up previously. Goland IDE seems to always auto add a dependency to the first group list after the Golang std libs 🤔

Reply:
fyi, i had opened a PR to add the openshift goimports to keep module grouping consistent https://github.com/Kuadrant/kuadrant-operator/pull/415. Backed it out as we seem to already use `goimports` in CI tests, but maybe we could look into it more.

Reply:
+1